core/fileutil/dir_windows.go

```diff
@@ -0,0 +1,10 @@
+package fileutil
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func DefaultDir() string {
+	return filepath.Join(os.Getenv("LOCALAPPDATA"), "Chain")
```

Comment:
Do you think it would be a good idea to put the word "core" somewhere in this path? I realize we are perhaps somewhat constrained because the installer maybe puts certain things here already. But we could add a subdirectory called `Chain Core` or `Core`. WDYT?